src/cephadm/cephadm

```diff
@@ -4682,7 +4681,7 @@ def wait_for_mon(
                              timeout=timeout)
         return ret == 0
 
-    is_available(ctx, 'mon', is_mon_available)
+    is_available(ctx, 'mon', is_mon_available, 3)
```

Comment:
> Note: as part of this fix I'm reducing the maximum tries for mon from 15 (default) to 3. So at most cephadm will wait 3 min for the mon.

do we need to reduce the retry? seems like that could be an issue on resource limited hardware (e.g. Raspberry Pi)

Reply:
> > Note: as part of this fix I'm reducing the maximum tries for mon from 15 (default) to 3. So at most cephadm will wait 3 min for the mon.
> 
> do we need to reduce the retry? seems like that could be an issue on resource limited hardware (e.g. Raspberry Pi)

I expected 3 min to be enough to wait for the mon process. Could the mon process takes more in order to get up & running?

Reply:
@mgfritch I reverted the changes related to the retry and only kept the original fix (related to timeout).